components/reusable-components/pagination/StructurePagination.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+interface StructurePaginationProps {
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+}
+
+/* eslint no-unused-vars: "off" */
+const StructurePagination: React.FC<StructurePaginationProps> = ({
+  currentPage,
+  totalPages,
+  onPageChange,
+}) => {
+  const handlePrevious = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNext = () => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
```

Comment:
this should be in another PR. Also we should have pages, like 1,2,3,4,5 not next and previous